src/components/button.scss

```diff
@@ -19,3 +20,11 @@
 .btn--secondary {
   background-color: $secondaryColor;
 }
+
+@media #{$mobile-version} {
```

Comment:
> button styling: when I try to change the button styling using media query, it doesnt work. So I thought It had something to do with the fact that I created a component for buttons and therefore I tried to use the media query in the button.scss file but it doesnt work neither.

What do you mean it doesn't work? It works on my side :)

<img width="1491" alt="image" src="https://user-images.githubusercontent.com/14869087/181092572-a9cf04b0-aae8-46ed-baac-9c37d6c1f026.png">